src/main/java/ezvcard/VCard.java

```diff
@@ -108,6 +113,8 @@
  * Represents a vCard.
  * @author Michael Angstadt
  */
+@JsonSerialize(using = JCardSerializer.class)
```

Comment:
I am worried about the annotations here.  Will ez-vcard still work if a developer does not want to use the jCard functionality and excludes the jackson dependencies from the project?  Will there be a `ClassNotFoundException` when the developer tries to create a new `VCard`?